src/components/Analysis/Dashboard/Dashboard.tsx

```diff
@@ -47,41 +40,77 @@ const Dashboard = ({accuracy, data}: todoType) => {
             <img src={CalendarIcon} className='dashboard-icon'/>
             <span>14 may, 2022</span>
           </div>
-        </div>
-      </div>
-      <div className='row' style={{ height: 'calc(50vh - 50px)', maxHeight: 500 }}>
-        <div className='d-flex flex-column align-items-center col-sm-4 col-md-2 col-lg-2 pb-6'>
-          <div className='card p-3' >
-            <h4 className='text-center'>Average<br/>mood</h4>
-            <AverageValueDisplay
-              percentage={Math.floor(Math.random() * 101)}
-            />
+          <div className='d-flex gap-3'>
+            <button className='btn mb-0'>
+              <img src={CopyLinkIcon} className='dashboard-icon'/>
+              <span>Copy link</span>
+            </button>
           </div>
         </div>
-        <div className='col-sm-4 col-md-5 col-lg-5'>
-          <div className='card chart-small'>
-            {data && < ChartToggler
-              dataType={'attentionScore'}
-              data={data.framesArray}
-              accuracy={accuracy}
-              isBarChartOnInit={true}
-              type={ATTENTION}
-              color={colors.primaryRed}
-              isThumbnail={true}
-            />}
+      </div>
+      <div className='row pt-2 pb-2' style={{ height: '50%' }}>
+        <div className='col-6 fill-height'>
+          <div className='card d-flex flex-row'>
+            <div className='d-flex flex-column align-items-center ps-2 pt-4' >
+              <h4 className='text-center fs-6'>Average</h4>
+              <AverageValueDisplay
+                percentage={Math.floor(Math.random() * 101)}
+              />
+              <div className='mt-3'>
+                <div className='d-flex justify-content-between gap-3'>
+                  <h4 className='fs-6 text-start'>Max: </h4>
+                  <h4 className='fs-6 text-end'>{(Math.random() * 101).toFixed(0)}%</h4>
+                </div>
+                <div className='d-flex gap-3'>
+                  <h4 className='fs-6 text-start'>Min: </h4>
+                  <h4 className='fs-6 text-end'>{(Math.random() * 101).toFixed(0)}%</h4>
+                </div>
+              </div>
+            </div>
+            <div className='chart-small'>
+              {data && < ChartToggler
+                frames={frames}
```

Comment:
This is breaking GitHub actions, can you get rid of it? Checking the props of the old file below should be enough.